Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Require MFA for Admins.json

```diff
@@ -34,7 +34,7 @@
       "includeUsers": []
     }
   },
-  "description": "Requires that privileged administrator roles authenticate using MFA.",
+  "description": "What is it: Requires that privileged administrator roles authenticate using MFA. \n Simeon Notes: Protect your tenant, require MFA for accounts that have privileged access. Simeon's baseline includes all applications and users except those with the Azure AD role, 'Directory Synchronization Accounts'. \n Learn More: https://docs.microsoft.com/en-us/office365/enterprise/protect-your-global-administrator-accounts",
```

Comment:
**What does this do?**
Requires that privileged administrator roles authenticate using MFA. The baseline includes all applications and users except those with the Azure AD role 'Directory Synchronization Accounts'.

**Why should you use this?**
If you want to further protect your tenant by requiring MFA for accounts that have privileged access. 

**What is the end-user impact?**
Users with privileged access must authenticate using MFA for each log in. 

@metamenos privileged administrator roles or users with privileged access? Probably synonymous but want to be consistent 

**Learn more**
[Protect your Microsoft 365 global administrator accounts](https://docs.microsoft.com/en-us/office365/enterprise/protect-your-global-administrator-accounts)